docs/output.md

```diff
@@ -682,9 +682,10 @@ The principal output files are the same between Salmon and Kallsto:
   - `<pseudo_aligner>.merged.gene_counts.tsv`: Matrix of gene-level raw counts across all samples.
   - `<pseudo_aligner>.gene_tpm.tsv`: Matrix of gene-level TPM values across all samples.
   - `<pseudo_aligner>.gene_counts.rds`: RDS object that can be loaded in R that contains a [SummarizedExperiment](https://bioconductor.org/packages/release/bioc/html/SummarizedExperiment.html) container with the TPM (`abundance`), estimated counts (`counts`) and transcript length (`length`) in the assays slot for genes.
-  - `<pseudo_aligner>.merged.gene_counts_scaled.tsv`: Matrix of gene-level library size-scaled counts across all samples.
+  - `<pseudo_aligner>.merged.gene_lengths.tsv`: Matrix of average within-sample transcript lengths for each gene across all samples, which can be used as an offset for gene-level analysis.
+  - `<pseudo_aligner>.merged.gene_counts_scaled.tsv`: Matrix of gene-level library size-scaled estimated counts from abundances across all samples.
```

Comment:
Sorry, let's not bake the 'from abundances' in everywhere. I realise that's what the pseudo-aligners do, but it not necessary to spell that out here- the user just needs to know the content of the files.

Reply:
No worries; it's OK to be picky here. 

Just to get this straight though (in case I'm missing something): what I wanted to describe was that those files contain "counts", that were created by `tximport` adding back in length/library size information into the TPM values reported by e.g. `salmon`. 

I.e., from the tximport vignette:

> We could alternatively generate counts from abundances, using the argument countsFromAbundance, scaled to library size, "scaledTPM", or additionally scaled using the average transcript length, averaged over samples and to library size, "lengthScaledTPM". 

This as opposed to the gene_counts.tsv file, which contains gene-level aggregated counts (based on the transcript counts).

In any case, if you think it's clear as is, we can just omit those lines again.

Reply:
As I understand it, abundance is effectively what these tools produce by default (via probabilistic maths I don't completely understand), such that even the content of `gene_counts.tsv` will be an estimate based on the abundance.

So I think it's best to keep things simple.